sprint5/zadanie3.js

```diff
@@ -0,0 +1,23 @@
+/*Concatenate First and Last Name into One String
+Given two strings, firstName and lastName, return a single string in the format "last, first".
+
+Examples
+concatName("First", "Last") ➞ "Last, First"
+
+concatName("John", "Doe") ➞ "Doe, John"
+
+concatName("Mary", "Jane") ➞ "Jane, Mary"
+Notes
+Don't forget to return the result.
+If you get stuck on a challenge, find help in the Resources tab.
+If you're really stuck, unlock solutions in the Solutions tab.*/
+
+function concatName(firstName, lastName) {
+	return lastName + ', ' + firstName;
+	//return lastName.concat(', ', firstName);
+}
+
+/*Stworzyłam funkcję o nazwie "concatName". Jej argumenty nazywają się firstName, lastName.
+Instrukcja return stanowi treść właściwą tej funkcji. Funkcja ta pobiera argumenty, ktora
+łączy argumenty ciągów do łańcucha i zwraca nowy ciąg.*/
+
```

Comment:
`Instrukcja return stanowi treść właściwą tej funkcji` - Nie wiem co to znaczy :D Proszę to usunąć.
`Funkcja ta pobiera argumenty, która łączy argumenty ciągów do łańcucha i zwraca nowy ciąg.` - to proszę przerobić na: "Funkcja ta zwraca ...".